src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.4.1.0.cs

```diff
@@ -21,7 +21,10 @@
 
 public partial class ExpectedExceptionTests : ConditionalWcfTest
 {
+#if FULLXUNIT_NOTSUPPORTED
     [Fact]
+#endif
+    [WcfFact]
```

Comment:
Reviewers: a couple of these touches were cleanup when I noticed this file had been missed in updating from FactAttribute.  All the places that added the new IssueAttribute needed to move to WcfFactAttribute to work properly, so I updated the rest just as cleanup.